server/views/pages/prisonerIncentiveLevelDetails.njk

```diff
@@ -0,0 +1,180 @@
+{% extends "../partials/layout.njk" %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "components/datePicker/datePicker.njk" import datePicker %}
+
+{% set title =  "Incentive details" %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Home",
+        href: '/'
```

Comment:
minor point – you can leave all of these as-is!
for the sake of consistency, we try to use only `"` in templates (to match how html attributes are wrapped) and `'` in typescript/javascript/stylesheets (cos that's the style dps seems to go for). the theory being that it's a bit easier to read if only one type is used throughout a file.